rhel/SPECS/dbaas-tools.spec

```diff
@@ -16,10 +20,13 @@ Summary:        A set of tools for Percona DBaaS
 License:        ASL 2.0
 URL:            https://github.com/kubernetes-sigs/aws-iam-authenticator
 Source0:        https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/%{commit_aws}/aws-iam-authenticator-%{shortcommit_aws}.tar.gz
+Source1:        https://github.com/kubernetes/kubernetes/archive/%{commit_k8s}/kubernetes-%{shortcommit_k8s}.tar.gz
 
 %if %{install_golang}
 BuildRequires:   golang >= 1.13.0
 %endif
+BuildRequires: go-bindata
```

Comment:
`go-bindata` is somewhat an infamous Go project. We shouldn't install that RPM if we can.
This module is in kubectl dependencies, so I guess it will install it during `go build`?

Reply:
This package is only required for the building purposes and won't be used anywhere except `%build`

Reply:
Yeah, I understand all that. The problem is that the original go-bindata project was removed and replaced by different one with the same name, so we can't be sure what version of go-bindata is packaged as RPM.

Also, is it really needed? I'm pretty sure that k8s vendors all generated files, including files created by go-bindata. If not please check if it can be installed by k8s build scripts.

Reply:
@AlekSi yes it is needed you can check makefile

Reply:
It's needed to build as it requires it:
```
+ cd /home/builder/rpm/BUILD/kubernetes-ec6eb119b81be488b030e849b9e64fda4caaf33c/
++ pwd
+ export GOPATH=/home/builder/rpm/BUILD/kubernetes-ec6eb119b81be488b030e849b9e64fda4caaf33c
+ GOPATH=/home/builder/rpm/BUILD/kubernetes-ec6eb119b81be488b030e849b9e64fda4caaf33c
+ cd src/github.com/kubernetes/kubernetes-ec6eb119b81be488b030e849b9e64fda4caaf33c
+ make WHAT=cmd/kubectl
+++ [0904 09:37:57] Building go targets for linux/amd64:
    ./vendor/k8s.io/code-generator/cmd/deepcopy-gen
+++ [0904 09:38:05] Building go targets for linux/amd64:
    ./vendor/k8s.io/code-generator/cmd/defaulter-gen
+++ [0904 09:38:09] Building go targets for linux/amd64:
    ./vendor/k8s.io/code-generator/cmd/conversion-gen
+++ [0904 09:38:16] Building go targets for linux/amd64:
    ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
+++ [0904 09:38:26] Building go targets for linux/amd64:
    ./vendor/github.com/go-bindata/go-bindata/go-bindata
Cannot find go-bindata.
make[1]: *** [gen_bindata] Error 1
make: *** [generated_files] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.jwCceD (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.jwCceD (%build)
```

Reply:
Looks like https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile.generated_files uses vendored go-bindata

Reply:
Yup, and that's why I removed it in: f85fd3e